ofnetBgp.go

```diff
@@ -671,18 +688,19 @@ func (self *OfnetBgp) modRib(path *api.Path) error {
 
 		// Install the endpoint in datapath
 		// First, add the endpoint to local routing table
-		self.agent.endpointDb[epreg.EndpointID] = epreg
+
 		err := self.agent.datapath.AddEndpoint(epreg)
 		if err != nil {
 			log.Errorf("Error adding endpoint: {%+v}. Err: %v", epreg, err)
 			return err
 		}
+		self.agent.endpointDb.Set(epreg.EndpointID, epreg)
```

Comment:
Endpoint was saved in epode so that data path can look it up if required.
good to keep it that way..